integration/mediation-tests/tests-common/admin-clients/src/main/java/org/wso2/esb/integration/common/clients/logging/LoggingAdminClient.java

```diff
@@ -19,18 +19,28 @@
 package org.wso2.esb.integration.common.clients.logging;
 
 import org.apache.axis2.AxisFault;
+import org.apache.axis2.client.Options;
+import org.apache.axis2.client.ServiceClient;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.wso2.carbon.logging.admin.stub.LoggingAdminStub;
+import org.wso2.carbon.logging.admin.stub.types.carbon.AppenderData;
+import org.wso2.carbon.logging.admin.stub.types.carbon.LogData;
 import org.wso2.esb.integration.common.clients.client.utils.AuthenticateStub;
 
+import java.rmi.RemoteException;
+
+/**
+ * This class can be used as the client to obtaining Log4J information about the system and also used for managing the
+ * system Log4J configuration.
+ */
 public class LoggingAdminClient {
     private static final Log log = LogFactory.getLog(LoggingAdminClient.class);
     private final String serviceName = "LoggingAdmin";
     private String endpoint = null;
     private LoggingAdminStub loggingAdminStub;
 
-    public static enum logLevel {OFF, TRACE, DEBUG, INFO, WARN, ERROR, FATAL}
+    public enum logLevel {OFF, TRACE, DEBUG, INFO, WARN, ERROR, FATAL}
```

Comment:
whats the best practice ? LogLevel or logLevel ?

Reply:
I am not sure what the standard is, but this was an existing code. I only removed the static term because it was useless. If I rename this enum I will have to change several classes irrelevant to this PR. I am also not sure if that would be okay as well.

Reply:
let's change it if there are not many classes